src/theme.css

```diff
@@ -7,5 +7,6 @@
   --button-color: white;
   --section-background-color: #f1f0f0;
   --project-font-family: sans-serif;
+  --darkmode-color: black;
```

Comment:
this naming is not clear, dark-mode is usually used in context where you want to change the whole theme of a website, and here it is being used just as contact bg color